transport-native-epoll/src/main/java/io/netty/channel/unix/DomainSocketChannel.java

```diff
@@ -16,6 +16,7 @@
 package io.netty.channel.unix;
 
 import io.netty.channel.socket.DuplexChannel;
+import java.io.IOException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.io.IOException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)